### PR TITLE
Next step button on attendee basic information page now submits the form.

### LINKED ISSUE #209: Clean up behavior of "next step button"
Right now, it sometimes looks like it should have submit button functionality when it does not. Also, it and the countdown should maybe not be at the top of the page. See https://github.com/littleweaver/django-brambling/commit/37f4f8a2263e#commitcomment-8032523 and nearby comments

### PR DESCRIPTION
**Pull request by harrislapiroff**
Instead of going to the next step, pressing "next step" submits the form. I believe this should be harmless, even if the form is unchanged, and will result in more predictable behavior. Partially resolves #209.

### CODE REVIEW

File: brambling/static/brambling/brambling.proxy.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * Documentation
+ * -------------
+ *
+ * Defines a declarative HTML syntax for triggering an event on one element
+ * by receiving an event on another element.
+ *
+ * E.g., To trigger form submission on form #myForm when hovering over a div:
+ *
+ * <div data-trigger="#myForm" data-on="mouseover" data-trigger="submit">
```
Comment (melinath) [inline brambling/static/brambling/brambling.proxy.js]: Two data-triggers?

Comment (harrislapiroff) [inline brambling/static/brambling/brambling.proxy.js]: Oops.

**Comment by nmorduch** [comment]
This looks cool. Can I assume Stephen vetted the JS?

**Comment by nmorduch** [comment]
:+1: Ready to merge pending answer to above comment

**Comment by harrislapiroff** [comment]
I'm actually superceding this with a pull request I am making now. Might try to tuck the JS somewhere else for future use though.

**Comment by nmorduch** [comment]
Ugh you're the worst <3

**Comment by melinath** [comment]
The JS looks fine FTR :-p

**Comment by nmorduch** [comment]
Closing without merging sounds like what's called for here.

**Pull request closed without merging**